config/routes.rb

```diff
@@ -7,8 +7,16 @@
   end
 
   resources :items do
+    collection do
+      get 'search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.